experimental/fluent/agentic/src/test/java/io/serverlessworkflow/fluent/agentic/LC4JEquivalenceIT.java

```diff
@@ -0,0 +1,288 @@
+/*
+ * Copyright 2020-Present The Serverless Workflow Specification Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.serverlessworkflow.fluent.agentic;
+
+import static io.serverlessworkflow.fluent.agentic.AgentWorkflowBuilder.workflow;
+import static io.serverlessworkflow.fluent.agentic.dsl.AgenticDSL.conditional;
+import static io.serverlessworkflow.fluent.agentic.dsl.AgenticDSL.doTasks;
+import static io.serverlessworkflow.fluent.spec.dsl.DSL.tasks;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import dev.langchain4j.agentic.AgenticServices;
+import dev.langchain4j.agentic.workflow.HumanInTheLoop;
+import io.serverlessworkflow.api.types.FlowDirectiveEnum;
+import io.serverlessworkflow.api.types.TaskItem;
+import io.serverlessworkflow.api.types.Workflow;
+import io.serverlessworkflow.api.types.func.CallTaskJava;
+import io.serverlessworkflow.api.types.func.ForTaskFunction;
+import io.serverlessworkflow.impl.WorkflowApplication;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class LC4JEquivalenceIT {
+
+  @Test
+  @DisplayName("Sequential agents via DSL.sequence(...)")
+  public void sequentialWorkflow() {
+    var a1 = AgentsUtils.newCreativeWriter();
+    var a2 = AgentsUtils.newAudienceEditor();
+    var a3 = AgentsUtils.newStyleEditor();
+
+    Workflow wf = workflow("seqFlow").tasks(tasks -> tasks.sequence("process", a1, a2, a3)).build();
```

Comment:
instead of `a1, a2, a3`, let's give the agents a name so users can grasp it better.